examples/ipv6only-app/esp32/main/main.cpp

```diff
@@ -0,0 +1,617 @@
+        }
+        if (request.bssid_count != 0)
+        {
+            scan_config.bssid = const_cast<uint8_t *>(reinterpret_cast<const uint8_t *>(request.bssid[0].bytes));
+        }
+        scan_config.channel     = request.channel;
+        scan_config.show_hidden = request.show_hidden;
+        scan_config.scan_type   = static_cast<wifi_scan_type_t>(request.active_scan);
+        if (request.active_scan)
+        {
+            scan_config.scan_time.active.min = request.scan_time_min_ms;
+            scan_config.scan_time.active.max = request.scan_time_max_ms;
+        }
+        else
+        {
+            scan_config.scan_time.passive = request.scan_time_min_ms;
+        }
+        esp_wifi_scan_start(&scan_config, false /* block */);
+        // Scan results will be sent to s_scanresult_writer once
+        // WIFI_EVENT_SCAN_DONE event is received
+        err = esp_event_handler_register(WIFI_EVENT, WIFI_EVENT_SCAN_DONE, &scan_event_handler, NULL);
+    }
+
+    pw::Status StopScan(ServerContext &, const chip_rpc_Empty & request, chip_rpc_Empty & response)
+    {
+        esp_wifi_scan_stop();
+        return pw::OkStatus();
+    }
+
+    pw::Status Connect(ServerContext &, const chip_rpc_ConnectionData & request, chip_rpc_ConnectionResult & response)
+    {
+        wifi_config_t wifi_config {
+        .sta = {
+            /* Setting a password implies station will connect to all security modes including WEP/WPA.
+             * However these modes are deprecated and not advisable to be used. Incase your Access point
+             * doesn't support WPA2, these mode can be enabled by commenting below line */
+	        .threshold = {
+                .authmode = static_cast<wifi_auth_mode_t>(request.security_type),
+            },
+
+            .pmf_cfg = {
+                .capable = true,
+                .required = false
+            },
+        },
+    };
+        memcpy(wifi_config.sta.ssid, request.ssid.bytes,
+               std::min(sizeof(wifi_config.sta.ssid), static_cast<size_t>(request.ssid.size)));
+        memcpy(wifi_config.sta.password, request.secret.bytes,
+               std::min(sizeof(wifi_config.sta.password), static_cast<size_t>(request.secret.size)));
+        PW_ESP_TRY(wifi_connect_sta(&wifi_config, &response));
+        return pw::OkStatus();
+    }
+
+    pw::Status Disconnect(ServerContext &, const chip_rpc_Empty & request, chip_rpc_Empty & response)
+    {
+        PW_ESP_TRY(esp_wifi_disconnect());
+        return pw::OkStatus();
+    }
+};
+
+} // namespace rpc
+} // namespace chip
+
+namespace {
+
+using std::byte;
+
+constexpr size_t kRpcStackSizeBytes         = (4 * 1024);
+constexpr uint8_t kRpcTaskPriority          = 5;
+constexpr size_t kScanResultsStackSizeBytes = (6 * 1024);
+constexpr uint8_t kScanResultsTaskPriority  = 3;
+constexpr size_t kTestStackSizeBytes        = (8 * 1024);
+constexpr uint8_t kTestTaskPriority         = 5;
+
+TaskHandle_t rpcTaskHandle;
```

Comment:
Need s_ prefix here ?

Reply:
Made the style consistent in the refactor